src/io/flutter/project/FlutterIconProvider.java

```diff
@@ -41,8 +41,10 @@ public Icon getIcon(@NotNull final PsiElement element, @Iconable.IconFlags final
       if (!file.isInLocalFileSystem()) return null;
 
       // Project root.
-      if (Objects.equals(file.getPath(), project.getBaseDir().getPath())) {
-        return FlutterIcons.Flutter;
+      if (project.getBaseDir() != null) {
```

Comment:
In case there's a race condition, maybe store `project.getBaseDir()` in a local?

    basedir = project.getBaseDir()

?

Reply:
makes sense, will do